controller/space_iterations_test.go

```diff
@@ -310,7 +307,7 @@ func (rest *TestSpaceIterationREST) TestWICountsWithIterationListBySpace() {
 		Name:       "Sprint 2.1.1",
 		SpaceID:    spaceInstance.ID,
 		Path:       append(childOfIteration2.Path, childOfIteration2.ID),
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c